test/integration/client-side-encryption/driver.test.ts

```diff
@@ -401,6 +413,176 @@ describe('Client Side Encryption Functional', function () {
       });
     }
   );
+
```

Comment:
There are some CSOT prose tests I think we need to implement too: https://github.com/mongodb/specifications/blob/master/source/client-side-operations-timeout/tests/README.md#3-clientencryption